src/components/ingests-page/expanded-content.tsx

```diff
@@ -97,7 +100,7 @@ export const ExpandedContent = ({
                               title={d.label}
                               style={{ marginRight: "1rem" }}
                             >
-                              {d.label.length > 40
+                              {d.label && d.label.length > 40
                                 ? `${d.label.slice(0, 40)}...`
                                 : d.label}
```

Comment:
```
{d.label && d.label.length > 40 ? `${d.label.slice(0, 40)}...` : d.label}
```
This will cause an error on first load if the manager doesn't add label to new audioDevices